From 809a1a0da5ed2953464d966e0920b3c38e886152 Mon Sep 17 00:00:00 2001
From: Stephane Danieau <stephane.danieau@st.com>
Date: Tue, 5 Dec 2023 11:21:38 +0100
Subject: [PATCH 0639/1141] arm64: defconfig: enable BT High speed and Low
 Energy features

Add Bluetooth High Speed (HS) features
Add Bluetooth Low Energy (LE) features

Change-Id: Ie588e38c9f6e807bd49fb5066eaee0aab01e6242
Signed-off-by: Stephane Danieau <stephane.danieau@st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 2 ++
 1 file changed, 2 insertions(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -4,6 +4,8 @@ CONFIG_ARM_SCMI_CPUFREQ=y
 CONFIG_ARM_SMC_WATCHDOG=y
 CONFIG_BACKLIGHT_CLASS_DEVICE=y
 CONFIG_BACKLIGHT_GPIO=y
+CONFIG_BT_HS=y
+CONFIG_BT_LE=y
 CONFIG_CAN_M_CAN=m
 CONFIG_CAN_M_CAN_PLATFORM=m
 CONFIG_COUNTER=m
